Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -164,6 +164,13 @@ var ListView = React.createClass({
      */
     renderSectionHeader: PropTypes.func,
     /**
+     * (props) => renderable
+     *
+     * A function that returns the scroll view in which the list rows are
+     * rendered. Defaults to returning a ScrollView with the given props.
+     */
+    renderScrollView: React.PropTypes.func.isRequired,
```

Comment:
Yeah, here's a small example:

```js
/**
 * @flow
 */
'use strict';

var React = require('react-native');
var { PropTypes, Text } = React;

var App = React.createClass({
  propTypes: {
    getContent: PropTypes.func,
  },

  render() {
    var content = this.props.getContent();
    return <Text>{content}</Text>;
  },
});
```

Flow prints:
```
/Users/ide/test/index.js:15:19,41: call of method getContent
Function cannot be called on possibly undefined value
/Users/ide/test/index.js:11:17,30: undefined
```

Adding isRequired makes the error go away.